### PR TITLE
Add documentation for copying projects.

### LINKED ISSUE #172: Documentation for Project Copying
As mentioned in 3blades/3blades#133, users will now be able to copy projects by sending a `POST` request to the projects endpoint with a specific payload, which is different than the currently defined payload for this endpoint. The documentation needs to reflect this.

### PR DESCRIPTION
**Pull request by johngriebel**
Closes #172 
Signed-off-by: John Griebel <johnkgriebel@gmail.com>

### CODE REVIEW

File: tbs_swagger.yml [hunk 1]
```diff
@@ -1404,6 +1404,69 @@ paths:
       summary: 'Delete a project'
       tags:
         - projects
+  /v1/{namespace}/projects/project-copy/:
```
Comment (raygesualdo) [inline tbs_swagger.yml]: Shouldn't this be `/v1/{namespace}/project-copy/`?

Comment (johngriebel) [reply tbs_swagger.yml]: I figured it should be under `projects` since it's a project operation.

Comment (raygesualdo) [reply tbs_swagger.yml]: Unless I'm reading this wrong though, that's now how it's implemented: https://github.com/3Blades/app-backend/blob/master/appdj/urls/unversioned.py#L90

Comment (johngriebel) [reply tbs_swagger.yml]: You're right. That's an issue with the app-backend PR. Nice catch.

**Pull request merged**